config.toml

```diff
@@ -24,3 +21,40 @@
 #
 # Default:
 #   reconnect_timeout = "10s"
+
+[turn]
+# Turn server IP address.
+#
+# Default:
+#   ip = "0.0.0.0"
+
+# Turn server port.
+#
+# Default:
+#   port = 3478
+
+# Static user on Turn server.
+#
+# Default:
+#   user = "USER"
+
+# Static user password on Turn server.
+#
+# Default:
+#   pass = "PASS"
+
+[redis]
```

Comment:
@tyranron ,

Зачем еще одно вложение? `[turn.redis]`

Reply:
@alexlapa because Redis potentially is one of the possible DBs for Coturn. If at any point in future we will support more, we won't introduce a breaking change in config.

Also, it's more readable/understandable that way what Redis is doing there.